src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al

```diff
@@ -0,0 +1,259 @@
+                        end;
+                    Status::Cancelled:
+                        begin
+                            AITTestMethodLine.SetRange("Test Suite Code", "Code");
+                            AITTestMethodLine.ModifyAll(Status, AITTestMethodLine.Status::Cancelled);
+                        end;
+                end;
+            end;
+        }
+        field(11; Tag; Text[20])
+        {
+            Caption = 'Tag';
+            DataClassification = CustomerContent;
+        }
+#pragma warning disable AA0232
+        // TODO: Remove pragma and add to SIFT
+        field(12; "Total Duration (ms)"; Integer)
+#pragma warning restore AA0232
+        {
+            Caption = 'Total Duration (ms)';
+            // ToolTip ='Specifies the Total Duration (ms) for executing all the tests in the current version.';
+            Editable = false;
+            FieldClass = FlowField;
+            CalcFormula = sum("AIT Log Entry"."Duration (ms)" where("AIT Code" = field("Code"), Version = field("Version"), Operation = const('Run Procedure'), "Procedure Name" = filter(<> '')));
+        }
+        field(13; Version; Integer)
+        {
+            Caption = 'Version';
+            Editable = false;
+            DataClassification = SystemMetadata;
+        }
+        field(14; "Default Min. User Delay (ms)"; Integer)
+        {
+            Caption = 'Default Min. User Delay (ms)';
+            InitValue = 100;
+            MinValue = 100;
+            MaxValue = 10000;
+        }
+        field(15; "Default Max. User Delay (ms)"; Integer)
+        {
+            Caption = 'Default Max. User Delay (ms)';
+            InitValue = 1000;
+            MinValue = 100;
+            MaxValue = 30000;
+
+            trigger OnValidate()
+            begin
+                if "Default Max. User Delay (ms)" < "Default Min. User Delay (ms)" then
+                    "Default Max. User Delay (ms)" := "Default Min. User Delay (ms)";
+            end;
+        }
+        field(16; "Base Version"; Integer)
+        {
+            Caption = 'Base Version';
+            DataClassification = CustomerContent;
+            MinValue = 0;
+            trigger OnValidate()
+            begin
+                if "Base Version" > Version then
+                    Error(this.BaseVersionMustBeLessThanVersionErr)
+            end;
+        }
+        field(19; RunID; Guid)
+        {
+            Caption = 'Unique RunID';
+            Editable = false;
+            DataClassification = SystemMetadata;
+        }
+        field(20; ModelVersion; Option) //TODO: Change this enum
+        {
+            Caption = 'AOAI Model Version';
+            OptionMembers = Latest,Preview;
+            DataClassification = SystemMetadata;
+        }
+        field(21; "No. of Tests Executed"; Integer)
+        {
+            Caption = 'No. of Tests Executed';
+            // ToolTip ='Specifies the number of tests executed in the current version.';
```

Comment:
The tooltips still need to be "activated".

Reply:
The current version of the compiler throws an error. We will uncomment after the bug is fixed